loopbackStarter/manifest.yml

```diff
@@ -0,0 +1,10 @@
+applications:
+- path: .
+  memory: 1G
```

Comment:
is 1G really necessary? free accounts get charged for using more than 512MB, and in the future it may even be less (256MB).